src/import.ml

```diff
@@ -65,6 +65,18 @@ module List = struct
       | None -> filter_map l ~f
       | Some x -> x :: filter_map l ~f
 
+  let filteri l ~f =
+    let rec filteri l i =
+      match l with
+      | [] -> []
+      | x :: l ->
+        let i = succ i in
```

Comment:
If I read this right, the index for the first element will be `1`. Shouldn't it be `0` as for `List.mapi`?

Reply:
FFS - thanks for catching that!

Reply:
(the use of `filteri` was assuming it was 0-based)